app/assets/scss/_brief_page.scss

```diff
@@ -0,0 +1,7 @@
+#incomplete-applications, #completed-applications {
```

Comment:
Is there reason why we can't use a class instead of ids?

Reply:
A class is preferable - the IDs were there as part of the stat block. It doesn't _look_ like those IDs are getting used by anything, but I kept them in just in case.

I've added a class - I could refactor and jettison the IDs if they're not necessary.